public/css/index.css

```diff
@@ -44,3 +44,7 @@ form.button {
 h4 {
     padding-top: 2em;
 }
+
+.donation-variable .form-group .btn {
```

Comment:
Is `.btn` used by a CSS framework or is that our own convention? If neither I'd opt for `button` over `btn`, we have the bytes to spare :wink: